dockstore-client/swagger.gradle

```diff
@@ -0,0 +1,49 @@
+dependencies {
+    // Version controlled by dependency management plugin
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
+    implementation 'org.glassfish.jersey.core:jersey-client'
+    implementation 'org.glassfish.jersey.media:jersey-media-json-jackson'
+    implementation 'org.glassfish.jersey.media:jersey-media-multipart'
+
+    implementation 'io.swagger.core.v3:swagger-annotations'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli'
+}
+
+def artifactGroup = "${group}.dockstore"
+
+tasks.register('downloadDockstoreYaml', Download) {
+    // this was the latest git commit hash at development time
+    src "https://raw.githubusercontent.com/dockstore/dockstore/bff1a2e0459f2d95faf0c8acfa575cf41e46e342/dockstore-webservice/src/main/resources/openapi3/openapi.yaml"
```

Comment:
I actually used this link so that we can pin the version for openapi.yaml file. It seems there is an openapi.yaml hosted at `https://dockstore.org/openapi.yaml`. Would that be better to use?

Reply:
Hmm, I'm not sure. Something fixed feels better than something floating. Something we keep as a copy feels _best_, if we check it into the repo ourselves, since we know it will never change... is that an option?

Reply:
Yeah I think checking it into the cbas repo should also work. It seems kind of best of both worlds for now so I will do that.

Reply:
I have checked-in the dockstore openapi.yaml file.